src/add_prefix.rs

```diff
@@ -52,6 +53,21 @@ where
     }
 }
 
+fn add_prefix_on_vehicle_journey_ids(
+    vehicle_journey_ids: &HashMap<(String, u32), String>,
+    prefix: &str,
+) -> HashMap<(String, u32), String> {
+    vehicle_journey_ids
+        .iter()
+        .map(|((trip_id, sequence), value)| {
+            (
+                (format!("{}{}", prefix, trip_id), *sequence),
```

Comment:
```suggestion
                (format!("{}:{}", prefix, trip_id), *sequence),
```
there's no : :thinking:  ?

Reply:
No
impl AddPrefix for VehicleJourney {
    fn add_prefix(&mut self, prefix: &str) {
        self.id = prefix.to_string() + &self.id;
        self.route_id = prefix.to_string() + &self.route_id;
        self.dataset_id = prefix.to_string() + &self.dataset_id;
        self.company_id = prefix.to_string() + &self.company_id;
        self.service_id = prefix.to_string() + &self.service_id;
        self.trip_property_id = self
            .trip_property_id
            .as_ref()
            .map(|id| prefix.to_string() + id);
        self.geometry_id = self.geometry_id.as_ref().map(|id| prefix.to_string() + &id);
    }
}

Reply:
Ok :+1: